### PR TITLE
modify Makefile  and add rhts-* command to test

### PR DESCRIPTION
**Pull request by qhsong**
In order to compatible with beaker, so I think out Makefile need to same as beaker file. 
If we need test our code to upstream, we need do other Makefile.Cause user didn't have `/usr/share/rhts/lib/rhts-make.include file`.Make will be exit.

### CODE REVIEW

File: lib/common.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```
Comment (qhsong) [inline lib/common.sh]: change it to `/usr/bin/env bash`

File: lib/log.sh [hunk 2]
```diff
@@ -19,7 +19,7 @@
 
 ((INCLUDE_LOG_SH)) && return ||INCLUDE_LOG_SH=1
```
Comment (qhsong) [inline lib/log.sh]: Notice: this can prevent double-time source error.

**Comment by qhsong** [comment]
@linuxqiao @sztsian @lulinqing @emmawu 
Maybe this is the first edition, Pls give some advice for it.

**Comment by sztsian** [comment]
I think before you integrate this with any kind of CI, you'd better talk with @lulinqing to see if we can utilize the current services.
For CI related information maybe we should talk internally instead of on Github

File: crash-sysrq-c/Makefile [hunk 3]
```diff
@@ -1,6 +1,19 @@
-# Makefile
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+#   Copyright (c) 2014 Red Hat, Inc. All rights reserved.
+#
+#   Author: CAI Qian    <caiqian@redhat.com>
+#   Update: Guangze Bai <gbai@redhat.com>
+#
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (linuxqiao) [inline crash-sysrq-c/Makefile]: I think you can change it to qhsong@redhat.com or remove this describe. Or update your name in here.

File: crash-sysrq-c/Makefile [hunk 4]
```diff
@@ -1,6 +1,19 @@
...
+
+export TEST        := /kernel/kdump/crash-sysrq-c
+export TESTVERSION := 3.0
```
Comment (linuxqiao) [inline crash-sysrq-c/Makefile]: This is reorg test case, package name and path will change, so set testversion := 1.0 is a good idea.

File: crash-sysrq-c/Makefile [hunk 5]
```diff
@@ -10,3 +23,14 @@ build:
...
 	./runtest.sh
 
+$(METADATA): Makefile
+	touch $(METADATA)
+	@echo "Name:          $(TEST)"                           > $(METADATA)
+	@echo "TestVersion:   $(TESTVERSION)"	                >> $(METADATA)
+	@echo "Path:          $(TEST_DIR)"                      >> $(METADATA)
+	@echo "TestTime:      1h"                               >> $(METADATA)
+	@echo "License:       GPLv3"                            >> $(METADATA)
+	@echo "Owner:         CAI Qian <caiqian@redhat.com>"    >> $(METADATA)
```
Comment (linuxqiao) [inline crash-sysrq-c/Makefile]: Please change Owner to " Kdump QE <kdump-qe-list@redhat.com>".

**Comment by linuxqiao** [comment]
I have a question, your branch is different from the master branch, it will be hard when we merger it to master. You should based on master branch.

**Comment by linuxqiao** [comment]
Another, you can follow "Issues" list. And add you issue in there.

**Comment by linuxqiao** [comment]
talk with qihan, merge.

File: crash-sysrq-c/Makefile (hunk 5 above)
Comment (sztsian) [inline crash-sysrq-c/Makefile]: I suggest to talk with @lulinqing first to confirm if it is proper to use this list

File: crash-sysrq-c/Makefile (hunk 3 above)
Comment (sztsian) [inline crash-sysrq-c/Makefile]: qsong, not qhsong

**Pull request merged**